packages/aws-cdk-lib/aws-events/lib/event-pattern.ts

```diff
@@ -263,6 +263,9 @@ export class Match implements IResolvable {
  *   300.0, and 3.0e2 are not considered equal.
  *
  * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEventsandEventPatterns.html
+ *
+ * For custom events, some optional properties are required. For more information, see
```

Comment:
I think this needs to be above the @see directive so it will be included alongside the other doc string content.

Reply:
Moved it